Documentation/contributor-guide/dependency_management.md

```diff
@@ -72,6 +72,18 @@ Please close the related PRs which were automatically opened by dependabot.
 When you bump multiple dependencies in one PR, it's recommended to create a separate commit for each dependency. But it isn't a must; for example,
 you can get all dependencies bumping for the module `go.etcd.io/etcd/tools/v3` included in one commit.
 
+#### Troubleshooting 
+In an event of bumping `grpc-*` dependencies, it might change `*.proto` file which can result in the following error:
```

Comment:
```suggestion
In an event of bumping the version of protoc, protoc plugins or grpc-gateway, it might change `*.proto` file which can result in the following error:
```